src/utils/lineGraph.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright © 2020 The PES Open Source Team pesos@pes.edu
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"image"
+	"sort"
+
+	drawille "github.com/cjbassi/gotop/src/termui/drawille-go"
+	ui "github.com/gizak/termui/v3"
+)
+
+// LineGraph implements a line graph of data points.
+type LineGraph struct {
+	*ui.Block
+
+	Data   map[string][]float64
+	Labels map[string]string
+
+	HorizontalScale int
+	MaxVal          float64
+
+	LineColors       map[string]ui.Color
+	DefaultLineColor ui.Color
+}
+
+func NewLineGraph() *LineGraph {
```

Comment:
Can you add a godoc type comment for exported entities such as this one?

Reply:
Done!